build/azure-pipelines/product-build.yml

```diff
@@ -143,8 +146,8 @@ trigger: none
 pr: none
 
 schedules:
-- cron: "0 5 * * Mon-Fri"
-  displayName: Mon-Fri at 7:00
+- cron: "0 6 * * Mon-Fri"
```

Comment:
@deepak1556 this scheduling rule change should not be part of the squashed commit as it was only done for exploration build